docs/kafka/access-mgmt-kafka/README.adoc

```diff
@@ -475,6 +477,37 @@ h|Operations
 |===
 --
 
+[id="proc-user-account-access_{context}"]
+== Granting users access to view other user accounts
```

Comment:
I think since RBAC is the mechanism here and is kind of an important keyword, I suggest we include it in this heading, something like "Using RBAC to allow users to view other user accounts". Then everything that follows stays the same. Wdyt?

Reply:
Better. Thanks